### PR TITLE
fix(validation): improve dinghy validations

### PR DESCRIPTION
**Pull request by cristhian-castaneda**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -51,3 +51,5 @@ tags
 
 # End of https://www.gitignore.io/api/vim
 dinghy
+
+vendor
```
Comment (dogonthehorizon) [inline .gitignore]: You should be able to delete your `vendor` folder since we got rid of it

**Review by dogonthehorizon — COMMENTED** [review]
Which issue is this resolving? Is there a test we could add here?

File: .gitignore (hunk 1 above)
Comment (cristhian-castaneda) [reply .gitignore]: done ✅  !

**Comment by cristhian-castaneda** [comment]
> Which issue is this resolving? Is there a test we could add here?

The issue itself corresponds to a series of scenarios where the Validator is not working as expected.

**Review by dogonthehorizon — APPROVED** [review]

**Pull request merged**